### DISCUSSION TITLE
Future sustainability

### DISCUSSION THREAD

**Discussion by JPGuillemin** [fast4x/RiMusic#2323] [General]
Hi, 
RiM is by far the most mature, feature rich, and user friendly music player available today (I tested many, many, a lot). Thus it provides FLAC equivalent sound quality (opus is transparent above 128kbps).

So I just want to know how you plan to involve yourself in the project in the future : how many years can we expect to rely on RiMusic without having to switch to something else or hopefully find another fork ?

Thanks for the beautiful app
All the best

**Comment by KOLIWAN** [comment]
Sounds like a job interview:
Where do you see yourself in 5 years?

**Comment by fast4x** [comment]
Simple and realistic enough to say that in 5 years there will be many forks of RiMusic and everyone will always have something in common in terms of audio quality. The quality will always and exclusively depend on the content that Google gives us without asking for a mandatory subscription. I refer to the content you listen to from RiMusic and all the other forks that rely on the YT/YTM api.

RiMusic is a project made of collaborators who have in common the passion to listen to free music, not pirate and maybe not want to be tracked by various trackers that "rightly" a capital company like Google must put in the field. RiMusic is therefore neither a company nor a product to sell or compete with.

The quality of the music then goes into the background, if quality is all that you want today as in the future, RiMusic and the other forks of other projects that use the same music source, are not and will not be applications for "experts" of music.

This is my thought.

**Comment by JPGuillemin** [comment]
Just asking if you plan to support the app in the long term.

**Reply by fast4x** [reply]
Yes

**Comment by JPGuillemin** [comment]
Thanks :)
About the quality : there's no reason for it to change, cause most people don't know that 16/44 PCM provide a better sound that 24/96 PCM, and that modern codecs like OPUS or AAC HE+ are PCM équivalent at around 128kbps. So streaming Platforms will keep providing paid Hires content subscriptions for most customers who ignore that the OPUS 160 is at least equivalent if not better than Hires.

**Comment by fast4x** [comment]
Nice info, thanks

**Comment by JPGuillemin** [comment]
24bits / 96Khz audio is useful for 2 use-cases : 
1) studio mixing & mastering (to have enough dynamic headroom for DSP processing); 
2) Transport from source to player (to have enough dynamic range for volume control).

But for the "source played" : it has been proven many times (and never disproved) than 24bits is useless ; and that 96KHz subsonic frequencies are just dirty artefact that create noise in the electronic by intermodulation and overdrive.

So : 
- never use Highres bullshit files
- always use a transport that run at 24 bits and above (LDAC and APTX-HD)
- with a DAC that suport volume control at 24 / 32 bits
